dotcom-rendering/src/web/components/PullQuoteBlockComponent.tsx

```diff
@@ -199,9 +258,11 @@ export const PullQuoteBlockComponent: React.FC<{
 			return (
 				<aside
 					css={[
-						decidePosition(role, design),
+						decidePosition(role, format.design),
 						css`
-							${headline.xxsmall({ fontWeight: 'bold' })};
+							${headline.xxsmall({
+								fontWeight: decideFontWeight(format.theme),
```

Comment:
I think because this is only used once, make sense to just have it inline (for font weights, rather than colours)